terms/dontKnow.jsonld

```diff
@@ -0,0 +1,9 @@
+{
+    "@context": {
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#"
+    },
+    "@type": "rdfs:Class",
+    "@id": "dontKnow",
```

Comment:
perhaps there should be a class called ResponseOptions and this could be of that type. for an example see: https://schema.org/Male.jsonld